.github/workflows/android.yaml

```diff
@@ -43,14 +47,18 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '17'
+      - name: 'Install Tools'
+        run: |
+          sudo npm i -g firebase-tools
+          echo "FIREBASE_TOOLS_VERSION=$(npm firebase --version)" >> $GITHUB_ENV
```

Comment:
discussion on how to share information between workflow steps, including syntax for command interpolation while setting an environment variable: https://stackoverflow.com/a/57989070/9910298

example output of command executed so you may verify it is valid for an environment variable (no spaces or similar):

```
mike@krakatoa:~/work/invertase/react-native-firebase (main) % yarn firebase --version
14.24.2
```